cpp/velox/operators/functions/RegistrationAllFunctions.cc

```diff
@@ -78,10 +78,10 @@ void registerFunctionOverwrite() {
 
 void registerFunctionForConfig() {
   const std::string prefix = "not_allow_precision_loss_";
```

Comment:
We might do not need this prefix. In Velox, they are registered with the simple name w/o a suffix, like 'add' and 'add_deny_precision_loss'.
https://github.com/facebookincubator/velox/pull/10383/files#diff-4baee9f82f9c347f753972415d345941d2c2a110b608d480b090650171da096bR678-R682